### PR TITLE
Update to Phoenix master with new Ecto

### PR DESCRIPTION
**Pull request by josevalim**

### CODE REVIEW

**Comment by josevalim** [comment]
This is for upcoming Phoenix. We still need to update the channel guides though.

File: H_ecto_models.md [hunk 1]
```diff
@@ -444,21 +444,20 @@ Once a user submits the form rendered from `new.html` above, the form elements a
...
-  if changeset.valid? do
-    Repo.insert!(changeset)
-
-    conn
-    |> put_flash(:info, "User created successfully.")
-    |> redirect(to: user_path(conn, :index))
-  else
-    render(conn, "new.html", changeset: changeset)
+  case Repo.insert(changeset) do
+    {:ok, _user} ->
```
Comment (lancehalvorsen) [inline H_ecto_models.md]: Can I just take a moment to say how much I love the new tuple-style return values? :heart:

**Comment by lancehalvorsen** [comment]
Thanks @josevalim! How immanent is the release? I can just merge this and not push to phoenixframework.org if it's going to be relatively soon. Otherwise I'll hold off.

File: H_ecto_models.md (hunk 1 above)
Comment (josevalim) [inline H_ecto_models.md]: Yes, you can! Thank you! :D

**Comment by josevalim** [comment]
It was blocking on me but it is all on @chrismccord's hands now. So I will let him answer. :zzz: time for me!

**Pull request merged**